packages/core/src/components/menu/_common.scss

```diff
@@ -26,27 +31,54 @@ $dark-menu-background-color: $dark-gray4 !default;
   color: inherit;
   user-select: none;
 
-  &:not(#{$disabled-selector})#{$hover-selector} {
+  &:hover {
+    background: $menu-item-color-hover;
+    cursor: pointer;
+    text-decoration: none;
+    color: inherit;
+  }
+
+  &:active {
+    background: $menu-item-color-active;
+  }
+
+  &#{$active-selector} {
     background: $pt-intent-primary;
     cursor: pointer;
     color: $white;
   }
 
   &#{$disabled-selector} {
+    // override global a:focus styles
+    outline: none;
+    background: inherit;
     cursor: not-allowed;
     color: $pt-text-color-disabled;
   }
 
   .pt-dark & {
```

Comment:
This should live in `_menu.scss` to be consistent with how we do it in other components (below `.pt-large`?). It's where we declare all the modifiers